hack/debug.sh

```diff
@@ -542,16 +542,29 @@ case "$1" in
     "")
 	if [ -z "$aos_master_service" ]; then
 	    echo "Usage:"
-	    echo "  [from master]"
+	    echo "  [From master]"
+	    echo "  Automatic:"
 	    echo "    $0"
 	    echo "  Gathers data on the master and then connects to each node via ssh"
 	    echo ""
-	    echo "  [from any other machine]"
+	    echo "  Manual:"
+	    echo "    $0 --master"
+	    echo "  Runs only master debugging on local machine"
+	    echo ""
+	    echo "  [From any node]"
+	    echo "  Automatic:"
 	    echo "    $0 MASTER-NAME"
 	    echo "  Connects to MASTER-NAME via ssh and then connects to each node via ssh"
 	    echo ""
-	    echo "  The machine you run from must be able to ssh to each other machine"
-	    echo "  via ssh with no password."
+	    echo "  Manual:"
+	    echo "    $0 --node"
+	    echo "  Runs only node debugging on local machine"
+	    echo ""
+	    echo "  In case of Automatic usage, the machine you run from must be able to ssh" 
+	    echo "  to each other machine via ssh with no password."
+	    echo ""
+	    echo "  In Manual usage you have to login manually to the machine, upload this script," 
+	    echo " and create the meta directory."
```

Comment:
It's more than that. You need to copy the contents of the meta directory from the `--master` run to each node before running the script on the node, and you have to run the script *from* the meta directory. There might be some other constraints I'm not thinking of too... the nodes might expect the `node/` subdir to already exist, etc. It's not simple to do (or to describe how to do), because we don't really expect people to do it by hand. Which is why it wasn't documented...

What is this PR for? Why do you feel that we need to document these options? Especially since this script is slowly being deprecated in favor of `oadm diagnostics NetworkCheck` anyway...

Reply:
it's related to this bug: https://bugzilla.redhat.com/show_bug.cgi?id=1448447